infra/core/host/container-apps.bicep

```diff
@@ -20,6 +22,7 @@ module containerAppsEnvironment 'container-apps-environment.bicep' = {
     logAnalyticsWorkspaceName: logAnalyticsWorkspaceName
     applicationInsightsName: applicationInsightsName
     daprEnabled: daprEnabled
+    vnetName: vnetName
```

Comment:
do you want to consider providing option for VNET internal ? Internal : Yes| no or a bool value? REf: https://learn.microsoft.com/en-us/azure/container-apps/networking?tabs=workload-profiles-env%2Cazure-cli#accessibility-levels